### PR TITLE
Various style fixes

### PR DESCRIPTION
**Pull request by mrueg**
* Replaced ” with " and ’ with ' since they are more common.
* Removed trailing spaces before linebreak
* Few changes to improve readability

### CODE REVIEW

File: OpenMetrics.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
 
-With OpenMetrics, we are cleaning up and tightening the specification with the express purpose of bringing it into IETF. We are documenting a working standard with wide and organic adoption while introducing minimal, largely backwards-compatible, and well-considered changes. As of 2020, dozens of exporters, integrations, and ingestors use and preferentially negotiate OpenMetrics already.
+With OpenMetrics, we are cleaning up and tightening the specification with the expressed purpose of bringing it into IETF. We are documenting a working standard with wide and organic adoption while introducing minimal, largely backwards-compatible, and well-considered changes. As of 2020, dozens of exporters, integrations, and ingestors use and preferentially negotiate OpenMetrics already.
```
Comment (brian-brazil) [inline OpenMetrics.md:5]: express is correct

File: OpenMetrics.md [hunk 2]
```diff
@@ -80,34 +80,34 @@ If more than one MetricPoint is exposed for a Metric, then its MetricPoints MUST
...
 
-Exposers SHOULD avoid names that could be confused with the suffixes that text format sample metric names use. 
+Exposers SHOULD avoid names that could be confused with the suffixes that text format sample metric names use.
 
 * Suffixes for the respective types are:
 * Counter: '_total', '_created'
 * Summary: '_count', '_sum', '_created', '' (empty)
-* Histogram: ‘_count', ‘_sum’, ‘_bucket', '_created'
-* GaugeHistogram: ‘_gcount’, ‘_gsum’, ‘_bucket’
+* Histogram: ‘_count', ‘_sum', ‘_bucket', '_created'
```
Comment (brian-brazil) [inline OpenMetrics.md:102]: This still doesn't look right

**Review by brian-brazil — COMMENTED** [review]
You didn't seem to get all of the non-standard quote marks.

File: OpenMetrics.md (hunk 1 above)
Comment (mrueg) [reply OpenMetrics.md]: Thanks, reverted!

File: OpenMetrics.md (hunk 2 above)
Comment (mrueg) [reply OpenMetrics.md]: Yep, looks like I missed some more. I went over the doc again and replaced the remaining ones.

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**